swc/extensions.bzl

```diff
@@ -32,11 +31,16 @@ def _toolchain_extension(module_ctx):
                     registrations[toolchain.name],
                 ))
             else:
-                registrations[toolchain.name] = toolchain.swc_version
-    for name, swc_version in registrations.items():
+                registrations[toolchain.name] = struct(
+                    swc_version = toolchain.swc_version,
+                    swc_version_from = toolchain.swc_version_from,
+                )
+
+    for name, registration in registrations.items():
         swc_register_toolchains(
             name = name,
-            swc_version = swc_version,
+            swc_version = registration.swc_version if registration.swc_version else None,
```

Comment:
Can we just assign without the `if/else/none`?

Reply:
Yep, fixed